server/webapp/WEB-INF/rails.new/webpack/views/dashboard/personalization_modal_widget.js.msx

```diff
@@ -121,6 +123,17 @@ const AlertContainer = {
   }
 };
 
+const ToolTip = {
+  view(vnode) {
+    return <span class="tooltip-outer-container">
+      <span class="tooltip-inner-container">
+        <i class="tooltip-hint" onclick={ (e) => { e.stopPropogation; e.target.nextSibling.classList.toggle("show"); }}/>
+        <span class="tooltip" role="tooltip">{vnode.attrs.tooltipText} <a href={vnode.attrs.linkUrl} target="_blank">{vnode.attrs.linkText}</a></span>
```

Comment:
It would probably be a better pattern to just allow the content to pass through via `vnode.children`, rather than parsing special attrs.